src/main/java/com/example/weatherwaveapi/model/response/weatherapi/weather/Weather.java

```diff
@@ -0,0 +1,11 @@
+package com.example.weatherwaveapi.model.response.weatherapi.weather;
```

Comment:
let's remove all subfolders from the weather folder and just leave ALL classes in the weather folder